### PR TITLE
update production.md

### PR DESCRIPTION
**Pull request by xp44mm**
block code : diff -> javascript

### CODE REVIEW

File: src/content/guides/production.md [hunk 1]
```diff
@@ -228,6 +228,6 @@ It is crucial to minimize your CSS for production. Please see the [Minimizing fo
 
 ## CLI Alternatives
 
-Some of what has been described above can also be achieved by using the command line. For example, the `--optimize-minimize` flag will include the `TerserPlugin` behind the scenes. The `--define process.env.NODE_ENV="'production'"` will do the same for the `DefinePlugin` instance described above. And, `webpack -p` will automatically invoke both those flags and thus the plugins to be included.
```
Comment (xp44mm) [inline src/content/guides/production.md:231]: i run webpack in cli, but:

```bash
# npx webpack --optimize-minimize
[webpack-cli] Unknown argument: --optimize-minimize
```
all those args is unknown.

**Comment by chenxsan** [comment]
`diff` is ok here.

File: src/content/guides/production.md (hunk 1 above)
Comment (chenxsan) [reply src/content/guides/production.md:231]: If you think this is a bug, please report it on webpack-cli repository, thanks.

Comment (xp44mm) [reply src/content/guides/production.md:231]: If the problem is not very easy to say, just leave it to the future.

**Comment by montogeek** [comment]
As @chenxsan said, those code snippets should be formatted using `diff`.

**Pull request closed without merging**